python/GangaLHCb/ganga-proxy.py

```diff
@@ -28,31 +28,11 @@
 
 global_env = None
 
-
-def get_env():
-    global global_env
-    if global_env is None:
-        global_env = {}
-        for k, v in os.environ.iteritems():
-            if not str(v).startswith('() {'):
-                global_env[k] = v
-            else:
-                this_string = str(v).split('\n')
-                final_str = ""
-                for line in this_string:
-                    final_str += str(os.path.expandvars(line)).strip()
-                    if not final_str.endswith(';'):
-                        final_str += " ;"
-                    final_str += " "
-                global_env[k] = final_str
-
-    return global_env
-
-if "GANGADIRACENVIRONMENT" not in get_env():
+temp_env = os.environ()
```

Comment:
Isn't `os.environ` just a dictionary? I don't think the brackets are needed and the `temp_env` overall seems a bit excessive as you can just use `os.environ` instead.